packages/react-tabs-example/package.json

```diff
@@ -3,9 +3,9 @@
   "version": "0.2.7",
   "private": true,
   "dependencies": {
-    "@hiendv/react-tabs": "^0.2.7",
+    "@hiendv/react-tabs": "*",
     "core-js": "^2.6.9",
-    "vue": "^2.6.10"
+    "react-scripts": "^3.1.2"
```

Comment:
Should we also add `react` and `react-dom`?

Reply:
probably, I'll add them later.